components/Participant.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { createRef, useEffect, useState } from 'react';
+
+const Participant = ({ gain, participant, isYou }) => {
+  const [videoRef] = useState(createRef<HTMLVideoElement>());
+
+  const playTrack = track => {
+    if (track.kind === 'audio') {
+      if (!isYou) {
+        // Don't play your own audio
+        var audio = new Audio();
+        audio.srcObject = postProcessAudioTrack(track);
+      }
+    } else if (track.kind === 'video') {
+      taggedLog('Creating new MediaStream');
+      taggedLog('track:', track);
+      const mediaStreamTrack = track.mediaStreamTrack;
+      const stream = new MediaStream();
+      stream.addTrack(mediaStreamTrack);
+      // @ts-ignore
+      videoRef.current.srcObject = stream;
+    } else {
+      taggedLog('non-audio or video track:', track);
+    }
+  };
+
+  useEffect(() => {
+    participant.tracks.forEach(publication => {
+      // sometimes it passes `null` (?!), so check if it's real
+      if (publication.track) {
+        playTrack(publication.track);
+      }
```

Comment:
```suggestion
    participant.tracks.forEach(({ track }) => {
        track && playTrack(track);
```

(I might have missed brackets in this suggestion, GH multi-line is weird)